src/Diagnostics.RuntimeHost/Utilities/FileHelper.cs

```diff
@@ -74,7 +74,7 @@ internal static bool IsDllorPdb(FileInfo fileInfo)
 
         internal static string SanitizeScriptFile(string val)
         {
-            var lines = val.Split(new[] { '\n' }).Where(p => p.StartsWith("#r ") || p.StartsWith("#load "));
```

Comment:
This was added to Sanitize the script file coming from Local Development.
For Auto-Complete and intellisense, we add a reference to our framework dlls in Local Development template using #load and remove the #load while compiling against the server.

Reply:
This will also remove the `#load` which is added by the detector author who wants to loads gists. Since the compiling service supports `#load`, we may not need to remove `#load` if we can provide a `../Framework/References/_frameworkRef.csx` in compiling service. Or we can only remove `#load "../Framework/References/_frameworkRef.csx"`.

Reply:
So for the first option, Cindy tried not removing the #load earlier but then the process looked for the exact path to find framework csx in the backend which was not present. So it has to be removed. I liked your idea of not needing to remove it but it might involve extra work right now which is not really in the scope of this PR. 

A quick fix can be the second option you suggested which is removing the #load which ends with _frameworkRef.csx